integration-tests/tests/list_test.go

```diff
@@ -44,7 +44,7 @@ func (s *listSuite) TestListMustPrintCoreVersion(c *check.C) {
 	expected := "(?ms)" +
 		"Name +Version +Rev +Developer *\n" +
 		".*" +
-		fmt.Sprintf("^%s +.* +%s + [0-9]+ +(canonical|sideload) *\n", partition.OSSnapName(c), verRegexp) +
+		fmt.Sprintf("^%s +.* +%s + [x0-9]+ +(canonical|sideload) *\n", partition.OSSnapName(c), verRegexp) +
```

Comment:
Remove the x altogether here too. It also comes from the store, so there's no need.

Also removed the "sideload" message from the right hand side.